pom.xml

```diff
@@ -905,8 +906,22 @@
           <trimStackTrace>false</trimStackTrace>
           <forkedProcessTimeoutInSeconds>1800</forkedProcessTimeoutInSeconds>
           <rerunFailingTestsCount>${testRetryCount}</rerunFailingTestsCount>
+          <useSystemClassLoader>false</useSystemClassLoader>
+          <argLine>-Djava.library.path=target/natives</argLine>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>${maven-failsafe-plugin.version}</version>
+        <executions>
```

Comment:
you don't need to add this "executions" flag, otherwise we are going to execute failsafe every time you run Maven.

it is automatically pre-configured, it makes sense to declare it only in order to set a version